sidebars.js

```diff
@@ -86,6 +86,7 @@ module.exports = {
       items: ['semgrep-app/getting-started-with-semgrep-app', 'semgrep-app/dashboard', 'semgrep-app/rule-board', 'semgrep-app/findings', 'semgrep-app/integrations', 'semgrep-app/editor', 'semgrep-app/notifications', 'semgrep-app/sso', 'semgrep-app/rbac', 'semgrep-app/pricing-and-billing'],
     },
     'playground',
+    'playground-old',
```

Comment:
I think linking to the old Playground from the new Playground’s doc page is enough and we don’t need to list the old Playground in the sidebar nav. What do you think?

Reply:
Good idea. I'll remove this.